app/mailers/comment_mailer.rb

```diff
@@ -0,0 +1,10 @@
+class CommentMailer < ApplicationMailer
+  def send_notification_email(comment_id)
+    @comment = Comment.find comment_id
+    @user = User.find @comment.parent.user_id
+    @parent_id = @comment.parent_id
+    @url = "https://www.rubygamedev.com/internal_posts/#{@parent_id}"
+
+    mail(to: @user.email, subject: "Notification comment")
```

Comment:
Let's set the subject to `"Someone has commented on your post!"`. However, it would be nice to use Rails i18n for this so that we can eventually support multiple languages (¿Hablas español? :smile: )